### ISSUE TITLE
Is there ability to make JOIN operation?

### ISSUE THREAD

**Issue by pavlochernovyy** [Pamblam/jSQL#20]
Hey @Pamblam nice job! Have couple questions, please take a look when you are be able to:

- Do you plan add API methods to support `JOIN` operations?
- Is there ability to use `JOIN` within `.query()` method?

Thanks!

**Comment by Pamblam** [comment]
Yes, the V-4 branch has limited support for joins, but I haven't had time to work on it much and it still needs work. If you're interested in contributing feel free to do so.
